resources/guest_configs/ci.config

```diff
@@ -0,0 +1,6 @@
+CONFIG_IKCONFIG=y
```

Comment:
if for some reason we need to disable a config option, would it work with this concat approach?

Reply:
Yep, both these styles work, and are normalized to the `is not set` style:

```
# CONFIG_SQUASHFS_ZSTD is not set
CONFIG_SQUASHFS_ZSTD=n
```

Reply:
just double checking. If somewhere in the "base" config there's something like:

```
CONFIG_XYZ=n
```

and later we past in the end of the file:

```
CONFIG_XYZ=y
```

will the latter override the former?

Reply:
yes, I think that's how it works

Reply:
I wrote "internal" script but what I really meant is that the script is shipped with the kernel source tree, so we could call it directly
```patch
- cat "$@" >.config
+ scripts/kconfig/merge_config.sh "$@"
```

Reply:
But isn't that internal to the kernel? I don't know if it is using `merge_config.sh` internally, but concatenation seems to work fine.

Reply:
> yes, I think that's how it works

Can we validate that to avoid surprises?